build.sbt

```diff
@@ -20,7 +20,7 @@ import Keys._
 import sbtassembly.AssemblyPlugin.autoImport._
 import com.typesafe.sbt.SbtGit.GitKeys.gitRemoteRepo
 
-val beamVersion = "2.4.0"
+val beamVersion = "2.5.0-SNAPSHOT"
```

Comment:
CI script will update this when building beam-master branch. We shouldn't change this.